cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -942,6 +967,7 @@ constexpr bool is_split_decode()
  * @param chunks List of column chunks
  * @param min_row Row index to start reading at
  * @param num_rows Maximum number of rows to read
+ * @param page_validity Vector of booleans indicating the validity of each page
```

Comment:
validity is an odd word for this, but I can't come up with a better name :\

Reply:
nvm, after looking at the effect it has, the name seems more appropriate. As you were.

Reply:
>validity is an odd word for this, but I can't come up with a better name :\
I agree, another valid name that I have thought of for this is `page_mask`.

Reply:
I think I will change it to `page_mask` then

Reply:
Updated!